diff --git a/tests/test_hs071.py b/tests/test_hs071.py
index f6ccb5b..67559e0 100644
--- a/tests/test_hs071.py
+++ b/tests/test_hs071.py
@@ -1,6 +1,7 @@
 """Test solution of problem HS71 from the Hock & Schittkowski collection"""
 
 # Standard Python modules
+import sys
 import unittest
 
 # External modules
@@ -205,6 +206,8 @@ class TestHS71(OptTest):
 
     @parameterized.expand(["SNOPT", "IPOPT", "SLSQP", "PSQP", "CONMIN", "NLPQLP", "ParOpt"])
     def test_optimization(self, optName):
+        if optName == "IPOPT" and sys.platform in ["win32", "cygwin"]:
+            raise unittest.skipTest("Skipping IPOPT on Windows")
         self.optName = optName
         self.setup_optProb()
         optOptions = self.optOptions.pop(optName, None)
diff --git a/tests/test_large_sparse.py b/tests/test_large_sparse.py
index 5f88088..6d7483d 100644
--- a/tests/test_large_sparse.py
+++ b/tests/test_large_sparse.py
@@ -7,6 +7,7 @@ Only testing with SNOPT, which supports sparse formats.
 """
 
 # Standard Python modules
+import sys
 import unittest
 
 # External modules
@@ -112,6 +113,8 @@ class TestLarge(OptTest):
         ]
     )
     def test_opt(self, optName, sparse):
+        if optName == "IPOPT" and sys.platform in ["win32", "cygwin"]:
+            raise unittest.skipTest("Skipping IPOPT on Windows")
         self.optName = optName
         self.setup_optProb(sparse=sparse)
         sol = self.optimize()
